Jenkinsfile

```diff
@@ -533,6 +533,7 @@ pipeline {
         Smoke_targets   = " check MIOpenDriver"
         NOCOMGR_flags   = " -DMIOPEN_USE_COMGR=Off"
         NOMLIR_flags    = " -DMIOPEN_USE_MLIR=Off"
+        NOCK_flags      = " -DMIOPEN_USE_COMPOSABLEKERNEL=Off"
```

Comment:
[R] Used only once and complicates the source.
```suggestion
```

BTW the leading space should not be necessary.

Reply:
Done